data/json/npcs/talk_tags.json

```
@@ -607,7 +710,12 @@
       "<fuck_you>, <name_b>",
       "<fuck_you>, <swear> <name_b>",
       "<name_b>",
-      "<swear> <name_b>"
+      "<swear> <name_b>",
+      "You can fuck right off you <name_b>",
+      "I've had enough of you. Begone.",
```

```suggestion
      "I've had enough of you, begone",
```
Not really something someone would say in response to, say "could I have a little something extra?" Your changes in general here don't work for the snippet.